db/pf_version.sql

```diff
@@ -0,0 +1,15 @@
+
+SET @MAJOR_VERSION = 5,@MINOR_VERSION = 1, @PATCH_LEVEL = 0;
+SET @VERSION_INT = @MAJOR_VERSION << 16 | @MINOR_VERSION << 8 | @PATCH_LEVEL;
+
+--
+-- Table structure for table 'pf_version'
+--
+
+CREATE TABLE pf_version ( `id` INT NOT NULL, `version` VARCHAR(11) NOT NULL , PRIMARY KEY (id));
```

Comment:
When we spoke about the VERSION_INT I did not remember why it was done (I know a comment would have helped).
It was done to make we can order the version properly when we have double digit major/minor/patch numbers.

If we depend on the string value alone then we will have the following order for the version numbers.

5.1.0
5.10.0
5.2.0
5.3.0
5.4.0
5.5.0
5.6.0
5.7.0
5.8.0
5.9.0

The VERSION_INT is constructed in such a way where the version can be sorted properly. 
(As long as no major/minor/patch numbers goes above 255).

When updating the new version number in the DB you just change the appropriate version variable.